ic-agent/src/agent/response_authentication.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) fn lookup_request_status<Storage: AsRef<[u8]>>(
     ];
     match certificate.tree.lookup_path(&path_status) {
         LookupResult::Absent => Ok(RequestStatusResponse::Unknown),
-        LookupResult::Unknown => Ok(RequestStatusResponse::Unknown),
+        LookupResult::Unknown => Err(LookupPathUnknown(path_status.to_vec())),
```

Comment:
To see if I understand:
- Before https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/13886, IC would return LookupResult::Unknown
- After, IC will return LookupResult::Absent

Note that Agent::wait() will exit immediately if lookup_request_status returns Err(LookupPathUnknown).  Does this mean we need to wait to publish an agent version with this change until the IC is running a replica that incorporates MR13886?

Reply:
Indeed, the above two bullet points are correct.

> we need to wait to publish an agent version with this change until the IC is running a replica that incorporates MR13886

yes, indeed